src/MediatR/Mediator.cs

```diff
@@ -40,6 +41,29 @@ public Task<TResponse> Send<TResponse>(IRequest<TResponse> request, Cancellation
             return handler.Handle(request, cancellationToken, _serviceFactory);
         }
 
+        public Task<object> Send(object request, CancellationToken cancellationToken = default)
+        {
+            if(request == null)

```

Comment:
Write the ifs with an empty space so the code style stays coherent. So
`if(request == null)` to `if (request == null)` and the same thing for the if below.